ztp/policygenerator/resources/acm-policy-du-platform-check-template.yaml

```diff
@@ -0,0 +1,185 @@
+#     Pao operator is available and ready
+#     kube-apiserver pod is running
+apiVersion: policy.open-cluster-management.io/v1
+kind: Policy
+metadata:
+  name: du-platform-check-policy
+  namespace: du-platform-check
+  annotations:
+    policy.open-cluster-management.io/categories: CM Configuration Management
+    policy.open-cluster-management.io/controls: CM-2 Baseline Configuration
+    policy.open-cluster-management.io/standards: NIST SP 800-53
+spec:
+  remediationAction: inform
+  disabled: false
+  policy-templates:
+    - objectDefinition:
+        apiVersion: policy.open-cluster-management.io/v1
+        kind: ConfigurationPolicy
+        metadata:
+          name: du-platform-check-policy-check
+        spec:
+          remediationAction: inform
+          severity: low
+          namespaceselector:
+            exclude:
+                - kube-*
+            include:
+                - '*'
+          object-templates:
+            - complianceType: musthave
+              objectDefinition:
+                apiVersion: machineconfiguration.openshift.io/v1
+                kind: MachineConfigPool
+                metadata:
+                  labels:
+                    pools.operator.machineconfiguration.openshift.io/master: ""
+                  name: master
+                spec:
+                  machineConfigSelector:
+                    matchLabels:
+                      machineconfiguration.openshift.io/role: master
+                status:
+                  readyMachineCount: 1
+                  updatedMachineCount: 1
+                  unavailableMachineCount: 0
+                  conditions:
+                    - type: Updated
+                      status: "True"
+                    - type: Updating
+                      status: "False"
+                  configuration:
+                    source:
+                      - apiVersion: machineconfiguration.openshift.io/v1
+                        kind: MachineConfig
+                        name: container-mount-namespace-and-kubelet-conf
+                      - apiVersion: machineconfiguration.openshift.io/v1
+                        kind: MachineConfig
+                        name: load-sctp-module
+                      - apiVersion: machineconfiguration.openshift.io/v1
+                        kind: MachineConfig
+                        name: 02-master-workload-partitioning
+                      - apiVersion: machineconfiguration.openshift.io/v1
+                        kind: MachineConfig
+                        name: 04-accelerated-container-startup-master
+                      - apiVersion: machineconfiguration.openshift.io/v1
+                        kind: MachineConfig
+                        name: 05-chronyd-dynamic-master
+                      - apiVersion: machineconfiguration.openshift.io/v1
+                        kind: MachineConfig
+                        name: 50-performance-openshift-node-performance-profile
```

Comment:
Just a note that follow-on work will dynamically populate these values with the specific names of MC objects that are included in the deployment.